Makefile

```diff
@@ -29,6 +29,8 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+OS=$(shell uname -s)
```

Comment:
I think this show be `kernel`

Reply:
linux and darwin are not OS

Reply:
`-s  -- print name of the operating system` help shows it is os name

Reply:
-s      Write the name of the `operating system implementation` to standard output.

Reply:
forget it, it's not important